README.md

```diff
@@ -14,12 +14,12 @@ Detailed documentation for arc42 can be [found here](https://docs.arc42.org/home
 
 ## Systems
 
-[Overview](./systems/overview/01-Introduction_and_Goals.md) provides a high level view of the whole Wikidata / Wikibase system landscape.
+[Overview](./Introduction_and_Goals.md) provides a high level view of the whole Wikidata / Wikibase system landscape.
```

Comment:
I find this very confusing, because we are on an "Overview" page in the "Overview" super-section and here an "Overview"-link points somewhere else.